plugins/nodes/src/js/types/SortDirection.tsx

```diff
@@ -0,0 +1,2 @@
+// TODO: once DL is in place, this will be included from there
```

Comment:
What is DL? Why do we need the todo comment?

Reply:
DL = Data Layer
I think I wanted to create a JIRA for that, but forgot it 🤔

Reply:
Please do and add the issue number or remove the comment